docs/yaml/yaml.go

```diff
@@ -114,6 +114,9 @@ func GenYamlCustom(cmd *cobra.Command, w io.Writer) error {
 		if _, ok := curr.Annotations["experimental"]; ok && !cliDoc.Experimental {
 			cliDoc.Experimental = true
 		}
+		if _, ok := curr.Annotations["deprecated"]; ok && !cliDoc.Deprecated {
+			cliDoc.Deprecated = true
+		}
```

Comment:
I think this part of the code is for the command itself, not for the flags. For flags, this part of the code will have to be updated (if we're gonna use annotations for this, but see my other comment); https://github.com/docker/cli/blob/a32cd16160f1b41c1c4ae7bee4dac929d1484e59/docs/yaml/yaml.go#L190-L202

Reply:
Oh right, good catch!

Reply:
I fixed that one, let’s see how the action folds in that other comment 😁